dspace-server-webapp/src/main/java/org/dspace/app/rest/converter/SubmissionFormConverter.java

```diff
@@ -106,25 +107,31 @@ private SubmissionFormFieldRest getField(DCInput dcinput, String formName) {
         inputRest.setRegex(dcinput.getRegex());
 
         if (!StringUtils.equalsIgnoreCase(dcinput.getInputType(), "qualdrop_value")) {
-            // value-pair and vocabulary are a special kind of authorities
-            String inputType = dcinput.getInputType();
-
-            SelectableMetadata selMd = new SelectableMetadata();
-            if (authorityUtils.isChoice(dcinput.getSchema(), dcinput.getElement(), dcinput.getQualifier())) {
-                inputRest.setType(
-                        getPresentation(dcinput.getSchema(), dcinput.getElement(), dcinput.getQualifier(), inputType));
-                selMd.setAuthority(getAuthorityName(dcinput.getSchema(), dcinput.getElement(), dcinput.getQualifier(),
-                        dcinput.getPairsType(), dcinput.getVocabulary()));
-                selMd.setClosed(
-                        authorityUtils.isClosed(dcinput.getSchema(), dcinput.getElement(), dcinput.getQualifier()));
-            } else {
-                inputRest.setType(inputType);
+            if (dcinput.isMetadataField()) {
```

Comment:
I would move this `if` one level up and merge it with the `if (dcinput.isMetadataField())` from below. This would make it more clear, that the `qualdrop_value` stuff in the `else` below, should only be done on a metadata field, too.

Reply:
I've modified the if statements a bit. In practice this will be the same, but it may make it a little easier to read

Reply:
Looks to be resolved. Marking as such.